frontend/src/components/welcome/pricing.js

```diff
@@ -67,7 +67,7 @@ const tiers = [
     ],
     buttonText: 'Sign up',
     buttonVariant: 'contained',
-    link: 'https://gitpay.me/#/login'
+    link: 'http://localhost:8082/#/login'
```

Comment:
This should be the original value when we merge this
```suggestion
    link: 'https://gitpay.me/#/login'
```